runtime/v2/manager.go

```diff
@@ -26,6 +28,7 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdkmodule "github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/genutil/types"
```

Comment:
To remove this dependency a genesis themed extension interface in `x/genutil` could be used to return `[]json.RawMessage` for server v2.